internal/fs/utils.go

```diff
@@ -32,6 +32,26 @@ func FindCFiles(dir string) []string {
 	return findFiles(dir, HasCExtension)
 }
 
+// FindFilesRecursive returns a list files from the whole directory subtree.
+func FindFilesRecursive(dir string) []string {
```

Comment:
Much better Name and Interface!

Reply:
you mean, I should look for a better name?